spring-boot-project/spring-boot-starters/README.adoc

```diff
@@ -205,4 +205,7 @@ do as they were designed before this was clarified.
 | https://alexo.github.io/wro4j/[Wro4j]
 | https://github.com/michael-simons/wro4j-spring-boot-starter
 
+| https://line.github.io/armeria/[Armeria]
```

Comment:
This file is ordered alphabetically, can you please move this entry accordingly?

Reply:
I fixed that :)